modules/popup_ui.js

```diff
@@ -42,6 +38,27 @@ const createPopUp = (movieiId) => {
               <label for="button"><button id="button" class="button1" type="submit">Comment</button></label> 
             </form>
         </div>`;
+
+  const getList = getElement(singlleMovie, '.comment__list');
+  CommentService.commentItems.forEach((el) => {
+    const li = document.createElement('li');
```

Comment:
For more clean code and less redundancy what if you write `li.innerHtml = <span></span` etc..?